app/views/import/fogbugz/new.html.haml

```diff
@@ -0,0 +1,45 @@
+- page_title "Fogbugz Import"
+%h3.page-title
+  %i.fa.fa-bug
+  Import projects from Fogbugz
+%hr
+
+= form_tag callback_import_fogbugz_path, class: 'form-horizontal', multipart: true do
+  %p
+    Follow the steps below to export your Fogbugz project data.
+    In the next step, you'll be able to select the projects you want to import.
+  %ol
+    %li
+      %p
+        Log into your Fogbugz account and set your filter to show All cases.
+      %p
+        =image_tag 'fogbugz_filter.png'
```

Comment:
The fogbugz API only returns the bugs that part of the user's current filter. Unfortunately it seems that through the API there is no way to create a filter, you can only select one.

I'm currently investigating if there is a builtin filter that is the same for all fogbugz users that includes all cases.